frontend/src/pages/settings/account/Account.tsx

```diff
@@ -1,15 +1,34 @@
 import React from 'react';
 
-import Box from '@material-ui/core/Box';
-import Divider from '@material-ui/core/Divider';
-import Grid from '@material-ui/core/Grid';
-import Typography from '@material-ui/core/Typography';
+import { useHistory } from 'react-router-dom';
+import { Box, Divider, Grid, Typography, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 
 import ContentHeader from '../../../components/ContentHeader';
 import PasswordForm from '../../../features/settings/account/PasswordForm';
 import SettingsMenu from '../../../features/settings/SettingsMenu';
+import { deleteAccountAPI } from 'src/features/account/accountAPI';
+import { useAppSelector, useAppDispatch } from '../../../app/hooks';
+import { selectLogin, logout } from '../../../features/login/loginSlice';
+
+const useStyles = makeStyles(() => ({
+  warning: {
+    backgroundColor: 'red',
+  },
```

Comment:
You main use the theme for this:

```
warning: {
    backgroundColor: theme.palette.error.main,
    color: theme.palette.error.contrastText,
  }
```

And a better name may be `danger` instead of `warning`

Reply:
Done